src/docs/get-started/test-drive/_vscode.md

```diff
@@ -44,4 +44,9 @@ contains a simple demo app that uses [Material Components][].
 [Quickly switching between Flutter devices]: https://dartcode.org/docs/quickly-switching-between-flutter-devices
 [status bar]: {% asset tools/vs-code/device_status_bar.png @path %}
 
+
+## Troubleshooting
+
+If you get an ```adb: error: failed to copy...``` go to the Android Studio AVD Android Virtual Device manager, select your device, and clear the cache by selecting the dropdown on the right and choosing "Wipe data". 
```

Comment:
There are also some errors with this edit. For example:
* Line length exceeds 80 chars
* Inline code fonting uses only one tick mark.
* UI text is shown in bold.